### ISSUE TITLE
f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#60]
**Description:**
Repeat of [f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz1_zmtop50.hf.001 (issue #56) ](https://github.com/NCAR/amwg_dev/issues/56) with bkg Kzz increaed to 3 m^2/s

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001

On github:
https://github.com/NCAR/amwg_dev/tree/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001

**Sandbox:**
Locally:
/glade/work/hannay/cesm_tags/cam6_3_041

On github (if available):
https://github.com/ESCOMP/CAM/tree/cam6_3_041
hash: 963ebdc

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001/atm/](https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001/atm/)

**Contacts:**
@JulioTBacmeister 
@adamrher 
@cecilehannay

**Comment by cecilehannay** [comment]
@adamrher: This run completed and diags are available.

**Comment by adamrher** [comment]
I think @JulioTBacmeister only pointed you to the advance_clubb_core.F90, but it was not stated that you have to grab all the other modules from the kzz_zmtop50 run as well (and sub in the advance_clubb_core). Was that the problem?

**Comment by JulioTBacmeister** [comment]
I thought this had been resolved, but yes that would be the origin of any
problems Cecile had.  I neglected to mention that all the files in the
previous SourceMods needed to be retained.

**Comment by adamrher** [comment]
@cecilehannay can you check this run? Something goofy is going on with the archive directory for this run:
/glade/scratch/hannay/archive/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001/atm/hist

It looks like it might be getting its wires crossed with the buggy run? There is a rogue 1989 file in there ... The run directory is yet to complete year 1980, so I suppose it will copy that over into the archive directory when 1980 finishes.

**Comment by cecilehannay** [comment]
Yes, it is a file from the buggy run that was still in the run directory
and got copied over by the archiving script. It will get overwritten later
on. But I will clean it up to make sure nobody looks at it in the meantime.
Thanks

**Comment by cecilehannay** [comment]
@JulioTBacmeister and @adamrher: Diagnostics are updated.  

https://webext.cgd.ucar.edu/FWscHIST/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop50.hf.001/atm/
